features/admin.tenants.v1/components/edit-tenant/edit-tenant-form.tsx

```diff
@@ -330,6 +369,31 @@ const EditTenantForm: FunctionComponent<EditTenantFormProps> = ({
                         onSubmit={ handleSubmit }
                         className="edit-tenant-form"
                     >
+                        <FinalFormField
+                            key="organizationName"
+                            width={ 16 }
+                            className="text-field-container"
+                            ariaLabel="organizationName"
+                            required={ true }
+                            data-componentid={ `${componentId}-organization-name` }
+                            name="name"
```

Comment:
Let's use specific values for `name` attribute, otherwise browsers will autofill these fields with end users' account credentials.

```suggestion
                            name="organizationName"
```

Reply:
`name` is the request payload value property name used. Hence, decided to go ahead with the current naming.